packages/cli-exec/src/exec.js

```diff
@@ -58,7 +58,8 @@ export const exec = command('exec', {
     log.warn('Percy is disabled');
   } else {
     try {
-      yield* percy.yield.start();
+      const execType = parentCommand.name === 'app:exec' ? 'app' : 'web';
```

Comment:
We use commands as separate plugin modules. Instead of adding this line here. Add it in plugin and just pass `execType` here [ instead of parentCommand ]